src/lib/components/BaseResponseElement/index.js

```diff
@@ -187,7 +189,9 @@ export class BaseResponseElement extends BaseElement {
         } else if (isSelected && !isCorrect) {
           option.setAttribute('data-submitted', '');
           this.disableOption(option);
-          this.deselectOption(option);
+          if (typeof this['deselectOption'] === 'function') {
```

Comment:
Where is this coming from? Can we set `this.deselectOption = /** @type {function(Node)} */ null` in the ctor to solve this?